From: Neil Horman <nhorman@redhat.com>
Date: Wed, 22 Sep 2010 14:13:00 -0400
Subject: [net] trace: fix sk_buff typo in network tracepoints
Message-id: <20100922141300.GA15430@hmsreliant.think-freely.org>
Patchwork-id: 28336
O-Subject: [RHEL5.6 PATCH] tracing: fix typo that got missed (bz 568614)
Bugzilla: 568614
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Amerigo Wang <amwang@redhat.com>

Hey all-
	Fujitsu just noted that the patch I did for bz 568614 had a typo in it,
in which I specified struct sk_buff as struct sk_buf.  This patch fixes it,
resolving bz 568614.
	On a side note, despite this typo, this patch still compiles and runs,
which I find rather terrifying.  I wonder if someone with more insight into the
tracing macros might offer a way to make such a typo break the compilation as it
would otherwise do if this happened outside of this macro usage.

Neil

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/trace/net.h b/include/trace/net.h
index 358a6f7..ff672f0 100644
--- a/include/trace/net.h
+++ b/include/trace/net.h
@@ -16,6 +16,6 @@ DEFINE_TRACE(net_dev_queue,
 	TPARGS(skb));
 
 DEFINE_TRACE(netif_rx,
-	TPPROTO(struct sk_buf *skb),
+	TPPROTO(struct sk_buff *skb),
 	TPARGS(skb));
 #endif
